newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/ExpressionAutocompletionsDisplayer.js

```diff
@@ -225,6 +226,46 @@ const DisplayedBehaviorAutocompletion = React.forwardRef(
   }
 );
 
+type ExpressionDocumentationProps = {|
+  expressionMetadata: gdExpressionMetadata,
+  i18n: I18nType,
+  parameterRenderingService: ParameterRenderingServiceType,
+|};
+
+const ExpressionDocumentation = ({
+  expressionMetadata,
+  i18n,
+  parameterRenderingService,
+}: ExpressionDocumentationProps) => {
+  return (
+    <Column noMargin>
+      <Text style={defaultTextStyle} size="body2">
+        {expressionMetadata.getDescription()}
+      </Text>
+      {mapVector(
+        expressionMetadata.getParameters(),
+        (parameter, index) =>
+          !parameter.isCodeOnly() &&
+          // This filters parameters that are implicit because of the context.
+          // Free functions have an instanceContainer as first parameter,
+          // their first object parameter are kept.
+          (index !== 0 || parameter.getType() !== 'object') &&
+          (index !== 1 || parameter.getType() !== 'behavior') && (
```

Comment:
Could these different checks be put in a dedicated function with a comment per line explaining why they are excluded?

Reply:
Agree! We had this kind of stuff in some files related to the events based extensions editors.